Plugins/FastJet/include/Acts/Plugins/FastJet/Jets.hpp

```diff
@@ -83,6 +84,76 @@ class TrackJetBuilder {
   fastjet::ClusterSequence m_clusterSeq{};
 };
 
+class TruthJetBuilder {
+ public:
+  explicit TruthJetBuilder(const Acts::Vector4& fm) { m_fourMomentum = fm; }
+  const Acts::Vector4 getTruthJetFourMomentum() const { return m_fourMomentum; }
+
+ private:
+  Acts::Vector4 m_fourMomentum{0., 0., 0., 0.};
+};
+
+class JetProperties {
```

Comment:
Why do we need this class?
The following methods are all valid for a jet object. 

Why do we need a jet and a jet property class? 
I think the following members could (and maybe should) be associated to a jet object, in my opinion.